### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by climbfuji**
- update README.md: remove warning statement at the top of the file, check out correct branch, update version numbers for EMC_post and UFS_UTILS
- update submodule pointer for EMC_post following the merge of https://github.com/NOAA-EMC/EMC_post/pull/106 (change version number from 8.0.0 to 1.0.0)
- update submodule pointer for UFS_UTILS following the merge of https://github.com/NOAA-EMC/UFS_UTILS/pull/88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,19 +74,9 @@ The [NCEPLIBS-external documentation](https://github.com/NOAA-EMC/NCEPLIBS-exter
...
-
-`cd NCEPLIBS`
-
-`git checkout release/public-v1`
-
-Since this project uses submodules they must be initialized by running:
+Clone the repository and its submodules:
 
-`git submodule update --init --recursive`
+`git clone -b ufs-v1.0.0 --recursive https://github.com/NOAA-EMC/NCEPLIBS`
```
Comment (kgerheiser) [inline README.md]: Should this be `release/public-v1`?

Comment (climbfuji) [reply README.md]: According to Arun, the branch will be deleted at some point, but the tag will stay (this PR will be retagged as ufs-v1.0.0).

**Comment by arunchawla-NOAA** [comment]
Please check with Wen and George on the version numbers. I think George is
referring it to as ufs-v1.0.0 and Post has also made a decision to go with
ufs-v1.0.0 . Again please confirm.

**Comment by climbfuji** [comment]
@WenMeng-NOAA @GeorgeGayno-NOAA can you please help clarify which version numbers we should use for the UFS public release? The VERSION files in EMC_post and UFS_UTILS say 8.0.0 and 1.1.1, but apparently those should both be 1.0.0?
```
heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat NCEPLIBS-post/VERSION
8.0.0
heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat UFS_UTILS/VERSION
1.1.1
```
Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
> 
> 
> @WenMeng-NOAA @GeorgeGayno-NOAA can you please help clarify which version numbers we should use for the UFS public release? The VERSION files in EMC_post and UFS_UTILS say 8.0.0 and 1.1.1, but apparently those should both be 1.0.0?
> 
> ```
> heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat NCEPLIBS-post/VERSION
> 8.0.0
> heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat UFS_UTILS/VERSION
> 1.1.1
> ```
> 
> Thanks!

@aerorahul  You added the version file.  What is that file documenting?

**Comment by WenMeng-NOAA** [comment]
@climbfuji For EMC_post repos, use ufs-v1.0.0. I added @fossell for double confirming.

**Comment by WenMeng-NOAA** [comment]
@aerorahul Will you update the VERSION file at EMC_post with 1.0.0?

**Comment by climbfuji** [comment]
I can make those updates for both EMC_post and UFS_UTILS.

**Comment by fossell** [comment]
@climbfuji @GeorgeGayno-NOAA - Confirming @WenMeng-NOAA comment,  UPP has decided to use tag "ufs-v1.0.0" of EMC_post for UFS release.  So version number is 1.0.0.

**Comment by climbfuji** [comment]
Associated PRs (need to be merged first, then need to update submodule pointers here):

https://github.com/NOAA-EMC/EMC_post/pull/106
https://github.com/NOAA-EMC/UFS_UTILS/pull/88

**Comment by aerorahul** [comment]
> > @WenMeng-NOAA @GeorgeGayno-NOAA can you please help clarify which version numbers we should use for the UFS public release? The VERSION files in EMC_post and UFS_UTILS say 8.0.0 and 1.1.1, but apparently those should both be 1.0.0?
> > ```
> > heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat NCEPLIBS-post/VERSION
> > 8.0.0
> > heinzeller-lt:NCEPLIBS-ufs-v1.0.0 dom.heinzeller$ cat UFS_UTILS/VERSION
> > 1.1.1
> > ```
> > 
> > 
> > Thanks!
> 
> @aerorahul You added the version file. What is that file documenting?

@GeorgeGayno-NOAA The file`VERSION` contains a string documenting the version of the package in the following syntax `MAJOR.MINOR.PATCH.TWEAK`.  This is standard package versioning practice.  The `VERSION` file removes the hard-wired version number from `CMakeLists.txt`.  This is the modern `cmake` practice.  Variants are seen where this file is also named `VERSION.cmake` or placed in a `cmake` sub-directory and accessed via `include` commands in the `CMakeLists.txt` file.

**Pull request merged**